### PR TITLE
Raises SNS events on completion of deployments

### PR DESCRIPTION
**Pull request by DavidIHunt**

### CODE REVIEW

**Comment by jake-cross** [comment]
The tests are currently failing. This is an issue with the changes made to validation of the message attributes.

File: server/modules/monitoring/DeploymentMonitor.js [hunk 1]
```diff
@@ -7,12 +7,14 @@ let fs = require('fs');
 let co = require('co');
 let ms = require('ms');
 let logger = require('../logger');
+let moment = require('moment');
```
Comment (jake-cross) [inline server/modules/monitoring/DeploymentMonitor.js]: When we add new modules to a file, could we opt for `const` until it's shown that the value needs to be mutable.

File: server/modules/monitoring/DeploymentMonitor.js [hunk 2]
```diff
@@ -36,8 +38,29 @@ module.exports = {
 
       yield activeDeploymentsStatus.map(
         activeDeploymentStatus => monitorActiveDeploymentStatus(activeDeploymentStatus)
+          .then((status) => {
+            if (status && status !== Enums.DEPLOYMENT_STATUS.InProgress) {
+              let startTime = moment(activeDeploymentsStatus.startTime).toISOString();
+              let endTime = moment.utc().toISOString();
```
Comment (jake-cross) [inline server/modules/monitoring/DeploymentMonitor.js]: Would a unix timstamp be a more flexible value to provide to consumers of this message?

Comment (DavidIHunt) [reply server/modules/monitoring/DeploymentMonitor.js]: I always preferred ISO strings as they're just as easily parsed by code and they're human readable as well.

File: server/modules/monitoring/DeploymentMonitor.js (hunk 1 above)
Comment (DavidIHunt) [reply server/modules/monitoring/DeploymentMonitor.js]: Yes I will change that now.

File: server/modules/monitoring/DeploymentMonitor.js (hunk 2 above)
Comment (jake-cross) [reply server/modules/monitoring/DeploymentMonitor.js]: 👍 Good point, well made. You're right that it's just preference.

**Pull request merged**